.github/workflows/release-embedding-sdk.yml

```diff
@@ -211,12 +211,16 @@ jobs:
 
       - name: Create a PR updating readme + published version, and changelog (using GitHub Metabase Automation account)
         run: |
+          # TODO: Remove this step once github includes the new gh CLI version 2.65.0
+          eval "$(/home/linuxbrew/.linuxbrew/bin/brew shellenv)"
+          brew install gh
```

Comment:
These commands were initially tested in [this run](https://github.com/metabase/metabase/actions/runs/12742148795).